src/components/AppComponent.tsx

```diff
@@ -209,16 +211,29 @@ const AppComponent: FC<Props> = props => {
     // mobile safari must be detected because empty and full bullet points in Helvetica Neue have different margins
     mobile: isTouch,
     android: isAndroid,
+    native: Capacitor.isNativePlatform(),
     'drag-in-progress': dragInProgress,
     chrome: /Chrome/.test(navigator.userAgent),
     safari: isSafari(),
   })
 
+  useEffect(() => {
+    if (Capacitor.isNativePlatform()) {
+      StatusBar.setStyle({ style: Style.Dark })
+      // Android only, set statusbar color to black.
+      if (Capacitor.getPlatform() === 'android') {
+        StatusBar.setBackgroundColor({
+          color: '#000000',
```

Comment:
You should use `colors.bg` to reference the background color from the theme instead of hardcoding it.